### PR TITLE
FLOE-508: Updated Infusion for responsiveness.

### PR DESCRIPTION
**Pull request by ghost**
Updated Infusion build which also includes the responsiveness (arrows in smaller/mobile screen sizes).

### CODE REVIEW

**Comment by jobara** [comment]
@grrrero it looks like the permissions for admin-options.php, inject-dependencies.php, set-defaults.php, readme.txt, template.html, ui-options.php, uio.css, and uio.js changed from 644 to 755. Could you please switch those back to 644?

**Comment by ghost** [comment]
@jobara Eep. Ok!

**Comment by jobara** [comment]
@grrrero could you also update the lib/infusion/Version.md file to include information about the pre-release build that this version of Infusion is from. You should be able to follow this as an example ( https://github.com/fluid-project/uio-wordpress-plugin/blob/v0.1/lib/infusion/Version.md ). 

Also it seems that clicking the arrows triggers navigation, which makes me think that this PR includes some code changes from https://github.com/fluid-project/infusion/pull/844. If that's the case we'll want that PR to be merged and this updated to Infusion master before being merged. Could you please confirm where the instance of Infusion used here comes from?

**Comment by jobara** [comment]
@grrrero you should also bump the plugin's version number https://github.com/fluid-project/uio-wordpress-plugin/blob/master/ui-options.php#L6

**Comment by jobara** [comment]
@grrrero The hide/show preferences bar appears below the sliding panel in the responsive mode.
screen shot 2017-09-18 at 12 54 47 pm

This is probably because the markup in https://github.com/grrrero/uio-wordpress-plugin/blob/master/template.html needs to be updated. See https://github.com/fluid-project/infusion/blob/master/demos/uiOptions/index.html#L84-L111 for an example of the current structure.

**Comment by ghost** [comment]
@jobara I've updated it, do you mind taking a look? Thanks!!

File: template.html [hunk 1]
```diff
@@ -1,14 +1,29 @@
-<!-- BEGIN markup for Preference Editor -->
-<div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
-    <!-- This is the div that will contain the Preference Editor component -->
-    <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+        <div class="flc-overviewPanel fl-overviewPanel-container"></div>
```
Comment (jobara) [inline template.html]: You don't need the overviewPanel div

**Comment by jobara** [comment]
@grrrero it looks like you've added in the JSON dependency files from Infusion. These should be stripped out.

**Comment by ghost** [comment]
@jobara Oh! Ok, try again?

**Comment by jobara** [comment]
merged @ 52aec532a1b05dbfd3b98202314c86a4ab3c89bf

**Pull request merged**